applications/facemask/README.md

```diff
@@ -13,7 +13,7 @@ A host edge device. We currently support the following:
 * Coral Dev Board
 * AMD64 node with attached Coral USB Accelerator
 * X86 node
-* Jetson TX2 (only available for image inference)
+* Jetson 
 
 #### Software
 
```

Comment:
Can you please mention Nvidia Docker Toolkit as a requirement for training and inference on X86 nodes with GPU?

Reply:
Also JetPack 4.4 for Jetson

Reply:
Sure